### ISSUE TITLE
Total install fail...

### ISSUE THREAD

**Issue by Harold19776** [PodcastGenerator/PodcastGenerator#387]
**System Information**
Podcast Generator Version: 3.1
Webserver: Nearlyfreespeech.net
PHP Version: 7.2,7.3, 7.4

**Podcast Generator Config**

```
INSERT THE **CENSORED** CONFIG HERE
``` 

**Description of your problem:**
Originally I was trying to upgrade from an earlier version of PG. That totally failed so I was just going to install fresh and copy over my media folder and feed.xml. I copied over the podcastgenerator folder to my server and I got the following errors. There were more than those shown but they were all similar.

![Screen Shot 2021-01-08 at 8 03 35 PM](https://user-images.githubusercontent.com/18432476/104079141-81b3a600-51ef-11eb-86f2-e0a1805fb66c.png)
![Screen Shot 2021-01-08 at 8 03 44 PM](https://user-images.githubusercontent.com/18432476/104079142-824c3c80-51ef-11eb-905e-12c1164f1d65.png)
![Screen Shot 2021-01-08 at 8 03 54 PM](https://user-images.githubusercontent.com/18432476/104079143-82e4d300-51ef-11eb-8cb8-4692f5014556.png)
![Screen Shot 2021-01-08 at 8 04 11 PM](https://user-images.githubusercontent.com/18432476/104079145-837d6980-51ef-11eb-8f63-617b79d0dc61.png)

Once copied over I went to the web address mysite/podcastgenerator and installed. After I chose a language I got this:

![Screen Shot 2021-01-08 at 8 08 29 PM](https://user-images.githubusercontent.com/18432476/104079146-837d6980-51ef-11eb-85c5-f626bd754122.png)

I was able to check the permissions on two of the folders. I couldn't find scripts.

![Screen Shot 2021-01-08 at 8 09 38 PM](https://user-images.githubusercontent.com/18432476/104079230-fa1a6700-51ef-11eb-9792-fd2cf2b05448.png)
![Screen Shot 2021-01-08 at 8 10 03 PM](https://user-images.githubusercontent.com/18432476/104079233-fbe42a80-51ef-11eb-8dfa-c3944f243545.png)

Please help me. If you need any other information please let me know what it might be. If it help I use nearlyfreespeech.net as a webserver.

**Comment by Go-rom** [comment]
You didn't mention the version of your previous PG installation

**Comment by emilengler** [comment]
Keep in mind that you can't upgrade from 2.7 -> 3.1
You will need to use 3.0.1 before

**Comment by Harold19776** [comment]
Sorry I kept forgetting to come back here even after the notifications...

So Since I had not real help I tried several things.  I tried installing 3.1 fresh and not upgrading. It still didn't work. I eventually wound up going back to 2.7 from a fresh install and it works fine. I keep getting an error message about something expiring in November of 2020 but I think it refers to Podcast Generators ability to make webpages for your episodes. Honestly I only use it to generate my RSS feed so I think I'm fine now thanks...

I figured out part of my problems it was my permissions. I didn't have them right. That solved some things but I still couldn't get 3.1 to work.
